pkg/util/constants/constants.go

```diff
@@ -28,4 +28,6 @@ const (
 	PVProvisionerGCEPD  = "kubernetes.io/gce-pd"
 	PVProvisionerAWSEBS = "kubernetes.io/aws-ebs"
 	PVProvisionerNone   = "none"
+
+	HTTPServerListenAddr = "0.0.0.0:8080"
```

Comment:
Just put this to where the listen happens.

Also listen to `127.0.0.1`? It is easy to open it up later, but hard to limit it back.